### ISSUE TITLE
Alternatives to Google maps?

### ISSUE THREAD

**Issue by datawookie** [gerrywalsh/routePlanning#2]
As you have observed, Google maps is a bit of a beast to work with. Plus, as of midway through the year, they no longer offer geocoding as a free service?

What are the alternatives to Google maps for doing geocoding?

**Comment by GerardWalsh** [comment]
True, it is no longer a free service. I think [HERE](https://developer.here.com) is a superior alternative, both monetarily and ease of use, and they also provide a free service below 250k monthly transactions with widespread industry adoption (Baidu, Samsung, AUDI).

**Comment by datawookie** [comment]
Indeed! https://wiki.openstreetmap.org/wiki/Nominatim is another good
option. I think that Google might be shooting themselves in the foot.

**Comment by GerardWalsh** [comment]
Looks like a great alternative
